package.json

```diff
@@ -27,10 +29,12 @@
 		"@angular/router": "^15.0.0",
 		"@nativescript/angular": "^15.0.0",
 		"@nativescript/core": "~8.4.0",
+		"@nativescript/nx": "^4.2.0",
```

Comment:
This shouldn't be needed here I don't believe? can likely remove this.

Reply:
For some reason, the repo doesn't build without those changes! may I have a misconfiguration or some think like this.

Reply:
Try removing all these that I mentioned and then run: “npm run setup “ dnd g